tests/database.test.js

```diff
@@ -1,26 +1,35 @@
 const test = require('tape');
-const dbReset = require('../src/model/database/db_build');
+const dbReset = require('../src/model/database/db_seed');
 const dbConnection = require('../src/model/database/db_connection');
 
-const { addUser, getUser } = require('../src/model/db_queries');
+const { addUser, getUser } = require('../src/model/user_queries');
+const { getSection, saveAboutMe, saveSymptoms, saveBackgrounds,
+  saveAppointments, saveClosing } = require('../src/model/form_queries');
+
+const { getGenerator, insertGenerator } = require('../src/model/queryGenerator');
 
 test('Insert user into database', (t) => {
   dbReset()
     .then(() => {
-      return addUser('jam', 'jam@gmail.com', 'password');
+      return addUser('jam', 'jam1@gmail.com', 'password');
+    })
+    .then((id) => {
+      t.equal(typeof id, 'number', 'Returns a number (the user id)');
+      return addUser('jon', 'jam1@gmail.com', 'password');
```

Comment:
I think this causes our travis build to fail. something to do with having to rollback a database transaction which errors maybe...
- https://stackoverflow.com/questions/10399727/psqlexception-current-transaction-is-aborted-commands-ignored-until-end-of-tra
- https://stackoverflow.com/questions/38558254/postgresql-catching-transaction-error-and-rollback